dojo/settings/settings.dist.py

```diff
@@ -1120,6 +1120,7 @@ def saml2_attrib_map_format(dict):
     'Hydra Scan': ['title', 'description'],
     'DrHeader JSON Importer': ['title', 'description'],
     'PWN SAST': ['title', 'description'],
+    'Blackduck Hub Scan': ['title', 'cve', 'component_name'],
```

Comment:
`cve` is not allowed as a parameter for deduplication anymore, it has been replaced by `vulnerability_ids`.

When you use `component name` name for deduplication, you should use `component_version` as well.